### PR TITLE
Updates manual test script.

### LINKED ISSUE #1836: Update the manual test script
### Descriptive summary

Update https://github.com/uclibs/scholar_uc/blob/develop/TEST-SCRIPT.md to test functionality that currently isn't covered.

Look over the app and find areas we are not currently covering in our manual tests.

Examples of tests to add:
* Cloud uploads
* Batch ingest
* Batch edit
* Transfering work ownership

### PR DESCRIPTION
**Pull request by scherztc**
Includes more of the application overrides in the manual test script.

Closes #1836

### CODE REVIEW

File: TEST-SCRIPT.md [hunk 1]
```diff
@@ -6,16 +6,71 @@ These tasks are usually performed on http://scholar-qa.uc.edu after a deployment
...
+
+### Submit Contact Form
+1. Verify that re-captcha is displaying properly.
+1. Verify that the form submits.
+
+### Welcome page
+1.  Create a new user.
+1.  Login as new user.
+1.  Verify that the welcome page displays.
+1.  Verfiy that you can opt out of welcome page.
```
Comment (hortongn) [inline TEST-SCRIPT.md]: Verfiy (spelling)

File: TEST-SCRIPT.md [hunk 2]
```diff
@@ -62,6 +131,7 @@ These tasks are usually performed on http://scholar-qa.uc.edu after a deployment
 1. Change your profile picture
 1. Add a highlighted work to your profile
 1. Remove the highlighted work from your profile
+1. Confirm title, department, uc affliation, email, alternate_email, alternate_phone, website,and blog
```
Comment (hortongn) [inline TEST-SCRIPT.md]: affliation (spelling)

File: TEST-SCRIPT.md [hunk 3]
```diff
@@ -78,9 +148,9 @@ These tasks are usually performed on http://scholar-qa.uc.edu after a deployment
 1. Verify Manydeposits is listed as a contributor
 1. Edit the work, add a file
 1. Remove Student Delegate from your proxies
-1. Log in as Student Delegate
-1. Verify Student Delegate does not have access to Manydeposit's work (BUG #56)
-1. Verify Student Delegate does not have access to your work (BUG #499)
+1. Log inas Student Delegate
```
Comment (hortongn) [inline TEST-SCRIPT.md]: inas (spelling)

File: TEST-SCRIPT.md [hunk 4]
```diff
@@ -96,6 +166,31 @@ These tasks are usually performed on http://scholar-qa.uc.edu after a deployment
 1. Grant a group edit rights
 1. Grant a group edit rights that does not exist in the system
 
+### Transfer Ownership
+1. Login as Professor Manydeposits
+1. Create a PRIVATE work 
+1. Login as Professor Nodeposits
+1. Confirm that you can not see work
```
Comment (hortongn) [inline TEST-SCRIPT.md]: can not (spelling)

File: TEST-SCRIPT.md [hunk 5]
```diff
@@ -96,6 +166,31 @@ These tasks are usually performed on http://scholar-qa.uc.edu after a deployment
...
+1. Login as Professor Manydeposits
+1. Use the Transfer Ownership tool in the Dashboard to transfer work
+1. Login as Professor Nodeposits
+1. Cofirm that you can see work in My Works
```
Comment (hortongn) [inline TEST-SCRIPT.md]: cofirm (spelling)

File: TEST-SCRIPT.md [hunk 6]
```diff
@@ -128,8 +223,45 @@ These tasks are usually performed on http://scholar-qa.uc.edu after a deployment
...
 
 ### Design and Layout
 1. Check for changes in the UC (black) menu bar
 1. Re-size browser window to test responsiveness 
 1. Test different browsers and platforms (Chrome, Firefox, IE, Android, iOS, etc.)
+
+### Email notifications
+1. Verify that an email gets sent on first login.
+1. Verify that an email gets sent when assigning and unassigning proxies.
+1. Verify that an email gets sent when assigining someone editor
```
Comment (hortongn) [inline TEST-SCRIPT.md]: assigining (spelling)

**Comment by scherztc** [comment]
@hortongn :  This is ready.

**Pull request merged**